### ISSUE TITLE
Navigation block: Sub menu background transparent when picking a suggested background color on Varia

### ISSUE THREAD

**Issue by yashitamittal11** [WordPress/gutenberg#42256]
### Description

When using the Navigation block with the Varia and its child themes, the background color selected from the block settings is not honored. It is because the specificity of the fallback menu background color is higher than the CSS specificity of the selected block color.

I expect the background color selected for the Navigation block to apply to the submenu.

### Step-by-step reproduction instructions

Steps to reproduce
1. Activate the Varia or any child theme
2. Add a Navigation block
3. Add a menu element and a submenu to it
4. From the sidebar, select a background color
5. Make sure it's one of the colors suggested (not a custom color)
6. Save and check in the front end

### Screenshots, screen recording, code snippet

![image](https://user-images.githubusercontent.com/81776920/155529880-c6ec6b88-efbb-43c3-92b5-d82800b5333e.png)

I created a PR to fix this issue, however it needs further discussion.
Link to the PR: https://github.com/WordPress/gutenberg/pull/42211

### Environment info

- WordPress version 6.0
- Gutenberg version 13.6.0

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by carolinan** [comment]
Hi!
In the navigation block, there is both a block called submenu, and one called page list, which has sub menus.
Can you clarify which block to use to reproduce this?
Can I assume that it is the page list, as that is the block that is updated in the PR?

Which overlay and submenu setting are enabled? As in, "Overlay menu, mobile", "Open on click" "Show arrow".

**Comment by carolinan** [comment]
Is the background color also set on the navigation block itself? I mean from the image it looks like there is an an orange color on the navigation block? Like this?:

<img width="300" alt="Navigation block color panel with two orange backgrounds selected" src="https://user-images.githubusercontent.com/7422055/177913987-03b67324-ac6c-4acd-9044-fef6c4079d29.png">

**Comment by yashitamittal11** [comment]
>In the navigation block, there is both a block called submenu, and one called page list, which has sub menus.
Can you clarify which block to use to reproduce this?

I tested the issue using the submenu inside the navigation block. I don't see an option to add a page list as a submenu in the navigation block. Here's a screen recording for your reference - 


https://user-images.githubusercontent.com/21078987/178248947-9368fe2b-3995-4559-98fc-66cea6b3ee62.mov


>Can I assume that it is the page list, as that is the block that is updated in the PR?

I updated the pagelist block as we figured the fallback style is coming from there.

>Is the background color also set on the navigation block itself? I mean from the image it looks like there is an an orange color on the navigation block? Like this?:

Yes, the background color is set on the navigation block. I also tried adding the background color to the submenu via the 'Submenu & Overlay background' option but it doesn't work.

>Which overlay and submenu setting are enabled? As in, "Overlay menu, mobile", "Open on click" "Show arrow".

'Overlay Menu = Always' => Background color displays fine

![mobile menu](https://user-images.githubusercontent.com/21078987/178249218-3be20f68-0ebc-4936-83a9-f504ddb96dbc.png)


I am able to reproduce the issue on the big screen if
'Overlay Menu = Off' or 'Overlay Menu = Mobile'  

The settings  "Open on click" and "Show arrow" didn't matter in my test. I am able to reproduce the issue with both settings enabled and disabled.

**Comment by yashitamittal11** [comment]
I checked the background color for the submenu is working fine in the default Twenty Twenty Two and Twenty Twenty One theme. Should we handle this change in the Varia theme if it is not impacting other themes? 

I noticed different themes are handling the colors differently.
For example - Twenty Twenty Two theme has an` !important `selector to override the fallback background color.

![2022](https://user-images.githubusercontent.com/21078987/178439825-c97a0445-984c-4fa5-a6ee-b5ffa15d701c.png)

Seedlet theme adds the extra [class] selector to override the default color. 

![seedlet](https://user-images.githubusercontent.com/21078987/181175940-5308f02e-ee1c-4d23-a24b-ccd7bff2413d.png)


@mtias @tellthemachines
